src/Entities/Pawn.cpp

```diff
@@ -428,9 +428,14 @@ void cPawn::HandleFalling(void)
 			TakeDamage(dtFalling, nullptr, Damage, Damage, 0);
 
 			// Fall particles
-			// TODO: Re-enable this when effects in 1.9 aren't broken (right now this uses the wrong effect ID in 1.9 and the right one in 1.8)
-			// int ParticleSize = static_cast<int>((std::min(15, Damage) - 1.f) * ((50.f - 20.f) / (15.f - 1.f)) + 20.f);
-			// GetWorld()->BroadcastSoundParticleEffect(EffectID::PARTICLE_FALL_PARTICLES, POSX_TOINT, POSY_TOINT - 1, POSZ_TOINT, ParticleSize);
+			GetWorld()->BroadcastParticleEffect(
```

Comment:
This function already exists by the way:

https://github.com/cuberite/cuberite/blob/31a11a6df4922b590a50a5ff3d3c00d42a45599d/src/Broadcaster.h#L11-L13

You have to get to it through a `cBroadcaster`
```cpp
GetWorld()->GetBroadcaster().BroadcastParticleEffect([...]);
```